docker-nersc/Dockerfile

```diff
@@ -149,13 +147,14 @@ RUN apt -y update && apt install -y apt-utils \
     libcfitsio-bin \
     libgoogle-glog-dev \
     libceres-dev \
+    imagemagick \
     # Remove APT files
     && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 COPY --from=builder /usr/local /usr/local
 COPY --from=builder /opt/conda /opt/conda
 
-RUN pip --no-cache-dir install cython ipython
+RUN pip --no-cache-dir install ipython Pillow healpy seaborn
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>`](https://app.codacy.com/manual/dstndstn/legacypipe/pullRequest?prid=4930373)